src/lib/components/Swap/SwapButton.tsx

```diff
@@ -1,6 +1,7 @@
 import { Trans } from '@lingui/macro'
 import { Token, TradeType } from '@uniswap/sdk-core'
 import { useERC20PermitFromTrade } from 'hooks/useERC20Permit'
+import useWrapCallback, { WrapErrorText, WrapType } from 'hooks/useWrapCallback'
```

Comment:
This will need to be refactored to return a receipt so that it can be tracked as a pending transaction. This is also not currently usable by the widget because it is coupled to app state.

Reply:
See #3172